pkg/cloud/aws/actuators/scope.go

```diff
@@ -149,30 +153,40 @@ func (s *Scope) Close() {
 		return
 	}
 
-	// Update the resource version and try again if there is a conflict saving the object.
-	if err := retry.RetryOnConflict(retry.DefaultBackoff, func() error {
-		s.V(2).Info("Updating cluster", "cluster-resource-version", s.Cluster.ResourceVersion)
-		s.Cluster.Spec.ProviderSpec.Value = ext
-		s.V(6).Info("Cluster status before update", "cluster-status", s.Cluster.Status)
-		latest, err := s.ClusterClient.Update(s.Cluster)
-		if err != nil {
-			s.V(3).Info("Cluster resource version is out of date")
-			// Fetch and update the latest resource version
-			newestCluster, err2 := s.ClusterClient.Get(s.Cluster.Name, metav1.GetOptions{})
-			if err2 != nil {
-				s.Error(err2, "failed to fetch latest cluster")
-				return err2
-			}
-			s.Cluster.ResourceVersion = newestCluster.ResourceVersion
-			return err
-		}
-		s.V(5).Info("Latest cluster status", "cluster-status", latest.Status)
-		s.Cluster.Status.DeepCopyInto(&latest.Status)
-		latest.Status.ProviderStatus = status
-		_, err = s.ClusterClient.UpdateStatus(latest)
-		return err
-	}); err != nil {
-		s.Error(err, "failed to retry on conflict")
-	}
-	s.V(2).Info("Successfully updated cluster")
+	// First set the provider spec
+	s.Cluster.Spec.ProviderSpec.Value = ext
+	old, err := s.ClusterClient.Get(s.Cluster.Name, v1.GetOptions{})
```

Comment:
Instead of doing a Get here, should we be storing the old value when we instantiate the scope instead?